src/vs/base/parts/tree/browser/loading.svg

```diff
@@ -14,8 +14,8 @@
     circle:nth-child(8) { animation-delay: 0.525s; }
 
     @keyframes ball {
-      from { opacity: 1; }
-      to { opacity: 0.3; }
+      0% { opacity: 1; }
```

Comment:
Are these ones not getting used or something? If they're in the VS Code then we should verify that and if it's affecting them too then submit a PR to their repo to fix that. (and if it's not affecting this one try to figure out why)

Reply:
that is the old tree, I remember you said only we are using it now. right?

Reply:
Yeah - looks like they actually have deleted it a while back from their code so guess this is fine.

Reply:
We might want to move this tree stuff into our code then to avoid confusion and possible issues during merges...